django_mongodb/compiler.py

```diff
@@ -395,20 +395,10 @@ def build_query(self, columns=None):
         return query
 
     def get_columns(self):
-        """
-        Return a tuple of (name, expression) with the columns and annotations
```

Comment:
Was the docstring already outdated? You didn't think a new one would be useful?

Reply:
I don't think so; the return value remains unchanged. I decided not to modify it because doing so would break other functions, such as subquery.

Long story short: the selection forces changes to the names of some returning columns. This behavior makes it difficult to track when a column's name changes. While I don't know if this was caused by collisions, there's no test to confirm it, and I can't imagine one either.

That said, as long as _make_result returns tuples, we can manage our own names (e.g., __annotation#), and everything should work fine.

So, the docstring is fine.

Reply:
Don't know why I deleted it. 😬

Reply:
Great, is everything broken it worked in my computer. 😬 let's fix the things.